### ISSUE TITLE
Can't submit final solution.

### ISSUE THREAD

**Issue by HaraldDobner** [Codewars/codewars.com#1971]
I am trying to submit the kata "Total increasing or decreasing numbers up to a power of 10" in C#.

My solution is running in 32 Miliseconds on my local computer. This is an absolutely acceptable solution.

The pretest was running green, but I can't submit the solution becaues it timed out after 12 Seconds. 

This is not the first time I can't submit a propper working solution because of a time out. Why is the code running on codewars so much slower? Is codewars running on a Raspberry? ;)

**Comment by FArekkusu** [comment]
I'd rather bet that you're trying to solve a kata requiring an optimal time complexity with a garbage solution. If you need help, ask a `question` in the kata discourse, but don't polute this repo with invalid issues.

**Comment by hobovsky** [comment]
> This is not the first time I can't submit a propper working solution because of a time out.

It's somewhat interesting definition of "proper" - "works for x=5, and if you wait a week or two, you will find out it works for x=100, too" ;)

> Why is the code running on codewars so much slower? Is codewars running on a Raspberry? ;)

My personal experience is somewhat opposite: usually, CW test suite runs ~2 times faster than on my local machine, but it varies between languages.

> My solution is running in 32 Miliseconds on my local computer. This is an absolutely acceptable solution.

Your solution runs in 32ms _for what `n`_? How long does it run for `n`=300? How long does it run for 40 test case with `n`=300?

Your question actually made me curious and I attempted the kata, My first two attempts worked for n<10 and took ~15 minutes for n ~= 100. I improved it and got to ~2s for complete test suite. Then I checked other solutions, and... man, I suck.
